todo.css

```diff
@@ -89,7 +93,7 @@
   border-bottom: none;/*線を消す*/
 }
 
-.btnWrap{
+.btn-wrap{
```

Comment:
I changed it from camelCase to kebab-case here as you were using kebab for everything else